walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -164,176 +121,141 @@ After the mobile app is registered, it is displayed on the console.
+=== Real time sync 
 
-== [Optional] Add security and Audit log configuration 
+. Select one instance of the app and open side panel 
+image::images/showcase.png[integration, role="integr8ly-img-responsive"].
+. Select `Manage Tasks`.
+. Create a new task using `+` icon.
+. Put any information into the New Task view and click on the create button
+. Check other instances of the app and you should see the same task appear.
+. Make some changes to the task in any instance of the app, and you should see the all the changes are automatically synced across all instances.
 
-To use additional sevices we need to perform additional binding in
-Mobile Developer Console.
+=== Offline Support
 
-===  Steps
-. Log in to the link:{route-mdc-server-host}[Mobile Developer Console, window="_blank"]
-. Click on the *showcase-app* card to display the configuration screen.
-. Click the *Mobile Services* tab.
-. Click *Bind to App* for *Data Sync* in the *Unbound Services* section.
-. Click Next on Binding screen. 
-. Bind the `Mobile Metrics` and `Identity Management` services to the app by clicking on the "Bind To App" buttons. Use the default binding options, and once the bindings are completed, you will see them in the `Bound Services` section. 
-. Now go back to the `Configuration` tab again, you will see the `mobile-services.json` file is updated with information about the bound services. If you expand the row for each service, you will see the links to each of the service. Feel free to click on the links and explore the dashboard for each of the service.
-. Copy `mobile-services.json` to some temporary location. We going to need it later
+. In order to demo the offline feature, we need to get the device into offline mode first.
+.. On a mobile phone this can be done by turning on airplane mode. 
+.. In a browser you can use the developer tool to simulate offline mode (For example, in Chrome, you can open it by pressing F12, or using Command+Option+I on macOS. Select the `Network` tab, and check `Offline`).
+. Create a new task in the showcase app. The task should be created and the `Offline Changes` button in the footer should contain one change.
+. Try make a few more changes by either editing existing tasks, or creating new ones.
+. You can review all the changes you have made while offline directly by clicking on `Offline Changes` button.
+. Check other instances of the app and you will not see any of the changes you have made because the current one is offline.
+. Restore connectivity, and once the client is online again, you should see all `Offline Changes` are synced to the server and appearing in other instances of the showcase app.
 
+=== Conflict resolution
 
-[time=35]
-== [Optional] Check the Data Synchonization Audit Logs
+. Make sure there are 2 instances of the app running.
+. In both instances of the app:
+.. Go offline
+.. Make some changes to the same task
```

Comment:
This may no longer trigger conflicts.

```suggestion
.. Make changes on name and description fields for the same tasks on two separate clients
```